recipes/gridfill/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.3.0" %}
+
+package:
+    name: gridfill
+    version: {{ version }}
+
+source:
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/ajdawson/gridfill/archive/v{{ version }}.tar.gz
+    md5: 4d26437253b77d756e5b37e516c63b51
+
+build:
+    number: 0
+    skip: True  # [win and py3k or win64]
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - mingwpy  # [win]
```

Comment:
If you mean `win64+py2k`, yes. We are good on `win32+py2k` (To be honest I got some extensions to build even on `win64`, that seems to be a distutils bug and not `mingwpy`, not sure if that was resolved.)